CodesmithProjects/TrailBlazers
# /api/trails path and /api/moreInfo path added to the server

## Overview

Issue Type


Description
Added backend server paths to respond to aforementioned URL's.
Deleted unused previous files and mock data

Ticket Item

FEATURE https://github.com/CodesmithProjects/TrailBlazers/pull/1: SVC
FEATURE https://github.com/CodesmithProjects/TrailBlazers/pull/2

Steps to Validate Feature

Go to 'https://www.postman.com/'
Click on 'Postman API'
Click on '+' tab to create a new request
Enter 'http://localhost:4000/api/trails/ followed by desired zip code
Click the "Send" button
Verify response object at Response section at the bottom of the page.
Previous behavior

A 404 error would appear for requests to the aforementioned paths.

Expected behavior

Expected to get response objects with information from trail APIs

Screenshots & Videos

Example screenshot

![bRSRG1M](https://user-images.githubusercontent.com/118780080/230736552-aa9879c9-ec3d-43d4-8482-bd637d5408c3.png)

Additional context

Added similar functionality to the /api/moreInfo/ path. A valid park ID is needed after the last forward slash.
